src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -231,6 +233,20 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
         );
     };
 
+    removeSections(blockSchemaTypeForm: Object) {
+        let filteredForm = {};
+        Object.keys(blockSchemaTypeForm).map((key) => {
```

Comment:
The `map` call does not really make sense here, since it returns a new array, which is not used. We could use `forEach` instead, or use `reduce`, which would avoid the `let` variable at all:

```javascript
return Object.keys(blockSchemaTypeForm).reduce((blockSchema, key) => {
    if (blockSchemaTypeForm[key]['type'] === 'section') {
        blockSchema = {...blockSchema, ...this.removeSections(blockSchemaTypeForm[key]['items']};
    } else {
        filteredForm[key] = blockSchemaTypeForm[key];
    }

    return blockSchema;
}, {});
```

(Untested, might need small adjustements)